gradio/routes.py

```diff
@@ -583,61 +580,36 @@ async def predict(
 
         @app.get("/queue/join", dependencies=[Depends(login_check)])
```

Comment:
The route names are a bit confusing now. `queue/data` joins the queue and `queue/join` gives you the data for the prediction lol. Can we switch?